### ISSUE TITLE
Uncertainties from Lorentz corrections?

### ISSUE THREAD

**Issue by SimonHeybrock** [scipp/essdiffraction#34]
The Lorentz correction factors $d$-dependent and/or $\theta$-dependent contributions into the data. The currently implementation does not include error bars, i.e., we assume it is exact. There is however significant and somewhat know uncertainty. For example, the error bars on $\theta$ are given (amongst other things) by the pixel/voxel size and the sample size. I think we should include this when such information is readily available.

**Comment by jl-wynen** [comment]
This requires quantifies these errors first. We have so far entirely ignored errors on coords. Is this a separate requirement that we should track?

**Comment by SimonHeybrock** [comment]
Not necessarily, this can be done without storing errors in coords, e.g., by having a provider that computes the errors on $\theta$, which might not be a simple function of other coords. We should wait for scientist input before taking any further steps.

**Comment by celinedurniak** [comment]
I would agree on including the uncertainties. 
But first we have to clarify where the implementation of the Lorentz correction should be done: reduction or analysis (during refinement using GSAS-II for example). 
I am still waiting for the reply of some instrument scientists.

**Comment by jl-wynen** [comment]
> Not necessarily, this can be done without storing errors in coords, e.g., by having a provider that computes the errors on θ, which might not be a simple function of other coords. We should wait for scientist input before taking any further steps.

I wasn't talking about storage but about determining the uncertainties. And also about other places where they factor in. We first need to have an understanding of how to do those things before we can handle them here.

**Comment by SimonHeybrock** [comment]
> > Not necessarily, this can be done without storing errors in coords, e.g., by having a provider that computes the errors on θ, which might not be a simple function of other coords. We should wait for scientist input before taking any further steps.
> 
> I wasn't talking about storage but about determining the uncertainties. And also about other places where they factor in. We first need to have an understanding of how to do those things before we can handle them here.

I would not put it like that. We *can* include any uncertainties we know of. There is no need to have a complete and holistic understanding of *all* contributions, etc. I don't see a real obstacle to handling them here.
